### DISCUSSION TITLE
Could WordDumb be configured to work with Kindle app on Android?

### DISCUSSION THREAD

**Discussion by sepsol** [xxyzz/WordDumb#27] [Q&A] [answered]
Will this be a feature that will ever be considered?

**Comment by xxyzz** [comment, marked as answer]
Added in v3.20.0. Please read the [document](https://github.com/xxyzz/WordDumb#how-to-use) of how to install adb.

**Comment by xxyzz** [comment]
It's feasible if adb rooted debugging is enabled on the phone.

**Comment by xxyzz** [comment]
Hi @sepsol, I add a commit to send files to Android via adb. Please read the [document](https://github.com/xxyzz/WordDumb#how-to-use) and the latest test plugin can be downloaded from [GitHub Actions Artifacts](https://github.com/xxyzz/WordDumb/actions/workflows/tests.yml).
